### ISSUE TITLE
PowerVu

### ISSUE THREAD

**Issue by joolzg** [gfto/tsdecrypt#7]
Any plans of adding powervu as a decoding option?

**Comment by joolzg** [comment]
OK I have added powervu ECM and EMM processing but i need a little help.

EMM, processed but you need a management DES key (in a file callled powervu.key), if you have this you will get any key updates and these will be written to a 2nd file powervu.auto

now powerVu has 8 viable streams, Video, Hid, Audio1,2,3,4 and a couple more, when processing the ECM i get the des keys for the 5 i want, and here lies my problem as I need to be able to decode the correct stream with the correct des key.

Where in the TS structure can i get an index so i can tie the video pid to the video des key etc

I am trying to do this in procesc.c:decode_buffer

joolz
ps. Also when receiving the stream AC3 streams are not marked as audio but as PES Private Streams. I can provide a small dump for you showing this problem.

**Comment by cooliobr** [comment]
Hi @joolzg , you make any progress in this case?

**Comment by joolzg** [comment]
@cooliobr

yes it works but you have to add the pids manually as I could not find a good way of automagically doing this.

It also does auto key grab as well.

**Comment by cooliobr** [comment]
Can you explain how i can do?



Att.

Rafael Machado
REGS IT LTDA.
ANTIK Business Representative BRASIL
"Believe in the future, believe in your success"
"Não existe patch para a estupidez humana"
"Nós somos o que pensamos (só não pense que você é um super-herói e não
tente voar)"

2017-01-17 19:02 GMT-02:00 Julian Gardner <notifications@github.com>:

**Comment by euacreations** [comment]
@joolzg 
Can you tell me how to use this please
